### PR TITLE
App seems to be working as expected

### PR DESCRIPTION
**Pull request by jjjhill**
Test the shit outta this.

### CODE REVIEW

File: src/components/ShopContent.vue [hunk 1]
```diff
@@ -231,104 +229,78 @@ export default {
...
-        this.$store.dispatch('updateOwnerBalances');
-      })
+      window.Cryptoz.deployed().then((instance) =>
+        instance.buyCard(cardAttributes.type_id, {from: this.coinbase, value:(cardAttributes.cost*1000000000000000000)})
+      )
     },
     getCardForFree : function(type_id){
       console.log("Claiming card:" + type_id);
 
       showPendingToast(this);
```
Comment (mchigit) [inline src/components/ShopContent.vue:245]: If we are showing a pending toast for this, I think we should also show a rejected/success toast. Or you can remove this pending

File: src/components/ShopContent.vue [hunk 2]
```diff
@@ -231,104 +229,78 @@ export default {
...
+      window.Cryptoz.deployed().then((instance) =>
+        instance.getFreeCard(type_id, {from: this.coinbase})
+      )
     },
     buyBoosters : function() {
       //Hide the modal
       this.$bvModal.hide("buy-boosters-modal");
 
       //Change buy button to pending.. or show some pending state
       this.showSpinner = 1;
```
Comment (mchigit) [inline src/components/ShopContent.vue:255]: is this `showSpinner` used anywhere? In this function we are setting it to 1 but never set it back to 0

**Review by mchigit — COMMENTED** [review]
Couple nit comments, overall code LGTM. Going to play around the app for a bit

**Comment by mchigit** [comment]
**Couple of other things I noticed:**
- When buying booster credit in shop, after balance is updated the "Cryptoz NFTs" number gets updated, but it's wrong:
image
It says I have 1 NFT but I have 10 in my crypt
- When buying booster credit and reject the transaction, `Pending confirmation....` doesn't disappear and no toast. 
- I thought the URL is `my-cryptoz-nfts` now, is that lost somewhere in the merge conflicts?
- Rejecting minting card doesn't have any toast (probably related to one of my comments above)
- 2 balance updated toast when sacrificing a card. Probably trigger an event that was captured by multiple components? Same thing happens with gifting
- My czxp balance seems wrong lool, it's also not getting updated for all transactions I make (opening booster, buy and open booster, sacrifice etc)
image
- Some cards can't be minted. Appears to be gas limit too high, but I'm not sure if this is related to your change. Try minting this card: 
image

File: src/components/ShopContent.vue (hunk 2 above)
Comment (jjjhill) [reply src/components/ShopContent.vue:255]: took it out in favor of toasts

**Review by ryanprice — APPROVED** [review]
Lets merge to add-bsc and the netlify will give us a test deploy to look at

**Pull request merged**